tests/Microsoft.Bot.Builder.TestBot.Json/Samples/GeneratedForm/luis/luis.settings.development.westus.json

```diff
@@ -0,0 +1,5 @@
+{
+    "luis": {
+        "sandwich_en-us_lu": "d221828d-3bac-48b9-a172-bc5dc24e7359"
```

Comment:
Thought it was tom only?

Reply:
this was generated by using bf luis:build, I ran to generate Development then created one for tomlm

---
In reply to: [376592396](https://github.com/microsoft/botbuilder-dotnet/pull/3350#discussion_r376592396) [](ancestors = 376592396)